rasa/core/channels/twilio_voice.py

```diff
@@ -0,0 +1,235 @@
+from sanic import Blueprint, response
+from sanic.request import Request
+from sanic.response import HTTPResponse
+from twilio.twiml.voice_response import VoiceResponse, Gather
+from typing import Text, Callable, Awaitable, List, Any, Dict, Optional, NoReturn
+
+from rasa.shared.utils.io import raise_warning
+from rasa.shared.core.events import BotUttered
+from rasa.shared.exceptions import RasaException
+from rasa.core.channels.channel import (
+    InputChannel,
+    CollectingOutputChannel,
+    UserMessage,
+)
+
+
+class TwilioVoiceInput(InputChannel):
+    """Input channel for Twilio Voice."""
+
+    SUPPORTED_VOICES = [
+        "man",
+        "woman",
+        "alice",
+        "Polly.Mads",
+        "Polly.Naja",
+        "Polly.Lotte",
+        "Polly.Reuben",
+        "Polly.Nicole",
+        "Polly.Russell",
+        "Polly.Amy",
+        "Polly.Brian",
+        "Polly.Emma",
+        "Polly.Amy-Neural",
+        "Polly.Emma-Neural",
+        "Polly.Brian-Neural",
+        "Polly.Raveena",
+        "Polly.Ivy",
+        "Polly.Joanna",
+        "Polly.Joey",
+        "Polly.Justin",
+        "Polly.Kendra",
+        "Polly.Kimberly",
+        "Polly.Matthew",
+        "Polly.Salli",
+        "Polly.Ivy-Neural",
+        "Polly.Joanna-Neural",
+        "Polly.Kendra-Neural",
+        "Polly.Kimberly-Neural",
+        "Polly.Sally-Neural",
+        "Polly.Joey-Neural",
+        "Polly.Justin-Neural",
+        "Polly.Matthew-Neural",
+        "Polly.Geraint",
+        "Polly.Celine",
+        "Polly.Mathieu",
+        "Polly.Chantal",
+        "Polly.Hans",
+        "Polly.Marlene",
+        "Polly.Vicki",
+        "Polly.Dora",
+        "Polly.Karl",
+        "Polly.Carla",
+        "Polly.Giorgio",
+        "Polly.Mizuki",
+        "Polly.Takumi",
+        "Polly.Liv",
+        "Polly.Jacek",
+        "Polly.Jan",
+        "Polly.Ewa",
+        "Polly.Maja",
+        "Polly.Ricardo",
+        "Polly.Vitoria",
+        "Polly.Camila-Neural",
+        "Polly.Cristiano",
+        "Polly.Ines",
+        "Polly.Carmen",
+        "Polly.Maxim",
+        "Polly.Tatyana",
+        "Polly.Conchita",
+        "Polly.Enrique",
+        "Polly.Miguel",
+        "Polly.Penelope",
+        "Polly.Lupe-Neural",
+        "Polly.Astrid",
+        "Polly.Filiz",
+        "Polly.Gwyneth",
+    ]
+
+    @classmethod
+    def name(cls) -> Text:
+        """Name of your custom channel."""
+        return "twilio_voice"
+
+    @classmethod
+    def from_credentials(cls, credentials: Optional[Dict[Text, Any]]) -> InputChannel:
+        """Load custom configurations."""
+        if not credentials:
+            return cls(initial_prompt="hello", assistant_voice="woman")
```

Comment:
that's covered by the lines after right? If you wanna handle the `is None` case you could also just do 
```suggestion
        credentials = credentials or {}
```